regenerate_fonts_file.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
This is the normal shebang for bash scripts, why did you change it?

Reply:
`/bin/bash` in not present on NixOS, so it didn't work on my system :upside_down_face: 
`/usr/bin/env bash` searches for the binary in the $PATH instead of assuming its location.

See:
* https://en.wikipedia.org/wiki/Shebang_(Unix)#Portability
* https://stackoverflow.com/questions/16365130/what-is-the-difference-between-usr-bin-env-bash-and-usr-bin-bash